docs/caching.rst

```diff
@@ -15,8 +15,11 @@ This variable can also be set through an environment variable as follows:
 
 .. code-block:: bash
 
-    $ export FUEL_LOCAL_DATA_PATH="/home/username/my_local_cache"
+    $ export FUEL_LOCAL_DATA_PATH="/LOCAL_PATH/my_local_cache"
 
 Please note that currently, caching is only implemented in the ``H5PyDataset``.
 In order to add caching to other types of datasets, one should use the
-``cache_file`` function, which is defined in ``fuel/utils/cache.py``.
+|cache_file|_ function.
+
+.. |cache_file| replace:: ``cache_file``
+.. _cache_file: api/utils.html#fuel.utils.cache.cache_file
```

Comment:
there is special syntax in theano doc that we use for that. Does FUEL support the same syntax?
```
    :func:`fuel.utils.cache.cache_file`
```
this remove the need to naem the html file itself.

Reply:
Done